boggle_blast.lic

```diff
@@ -0,0 +1,165 @@
+custom_require.call(%w[common common-healing common-items common-money common-travel drinfomon])
+
+class BoggleBlast
+  include DRC
+  include DRCH
+  include DRCI
+  include DRCM
+  include DRCT
+
+  def initialize
+    @stow_container = "backpack" # set this to be your main default storage container - i.e. the one used in STORE LIST set to default
```

Comment:
💡 These would be good candidates for `args`

Reply:
These would be good candidates, but given the temporary nature of this script I wasn't sure if it made sense to add them as such.  Your call, I'm fine either way.